modules/material/themes/material/silauth/loginuserpass.twig

```diff
@@ -100,9 +102,11 @@
             </span>
           </p>
 
-          <script>
-            ga('send', 'event', 'error', '{{ errorcode|e('js') }}', 'message', '{{ message|e('js') }}');
-          </script>
+          {% if analytics_tracking_id is not empty %}
```

Comment:
I feel like @forevermatt wanting an abstraction to make the sense of these "is not empty" clearer, even though I know what it is.